web-wallet/src/style/dusk-components/button.css

```diff
@@ -12,16 +13,67 @@
   border-width: var(--control-border-size);
   text-transform: uppercase;
   transform-origin: center center;
-  transition-delay: 0s;
-  transition-duration: 0.2s;
-  transition-property: background-color, border-color, color, transform;
-  transition-timing-function: ease-in-out;
+  transition:
+    0.2s ease-in-out background-color,
+    border-color,
+    color,
+    transform;
 }
 
 .dusk-button:not(.dusk-anchor-button) {
   cursor: default;
 }
```

Comment:
Yeah, I don't it's needed!

Reply:
If this just reinforces the default browser behaviour, it can be removed.

Reply:
> If this just reinforces the default browser behaviour, it can be removed.

Absolutely yes: I think I added last summer to override tailwind in a project and then forgot about it.